src/ROUTES.ts

```diff
@@ -1,5 +1,5 @@
-import {IsEqual, ValueOf} from 'type-fest';
-import CONST from './CONST';
+import {type IsEqual, type ValueOf} from 'type-fest';
```

Comment:
What if there are types and variables in an import statement? We would need two import statements then

Reply:
I don't mind any way. What do you guys prefer? I prefer the single import with nested mixed types and variables

Reply:
I prefer `import type` to match the same pattern we have been doing with `export type`, [example](https://github.com/Expensify/App/blob/main/src/components/LocaleContextProvider.tsx#L131-L133).

Would be nice to have both imports and exports with same ways of using it. WDYT @chrispader @blazejkustra  ?

Reply:
It's our decision right now. We can have export mixed too like this:

`export {type SomeType, someVariable}`

Reply:
we can go for the separate approach 👍

Reply:
I thought something like this would be possible:
```ts
import AnimatedStepContext, type {StepContext, AnimationDirection} from './AnimatedStepContext';
```

Turns out it's not @chrispader @fabioh8010 @hayata-suenaga 😒 I played around with the current settings and it actually allows both:
```ts
import AnimatedStepContext, {type AnimationDirection, type StepContext} from './AnimatedStepContext';
// and
import type {AnimationDirection, StepContext} from './AnimatedStepContext';
import AnimatedStepContext from './AnimatedStepContext';
```

```ts
export type {LocaleContextProps};
// and
export {type LocaleContextProps};
```

Which is not ideal if we want to enforce one or another, wdyt?

Reply:
I can confirm, that the second version is not detected as an error. From what i've seen, the `fixStyle` in the rule's option only applies when fixing not-typed imports. If there already is a `type` keyword, it will not move the `type` keyword out of the brackets...

Reply:
We can try to fix this rule upstream. Should we still merge this change to our codebase though?

Reply:
I'm okay with waiting, doesn't look like a urgent thing imo.